test/Chainweb/Test/RestAPI.hs

```diff
@@ -156,7 +156,7 @@ httpHeaderTests envIO cid =
                         let d = cur - x
                         assertBool
                             ("test failed because X-Server-Time is of by " <> sshow d <> " seconds")
-                            (d <= 1)
+                            (d <= 2)
```

Comment:
One second seemed a bit low and could cause sometimes cause fest failures on GitHub runners